### PR TITLE
Add different year to salary report account

### LINKED ISSUE #460: Salary one year, altinndato different year
Add a checkbox to  the salary employeereport, report accounts to tell if the row is a salary with the salary date is in one year and the altinn date in different year

Add AGA% and Feriepenge% to salary report account
Print AGA% for the kommune on accountplan for employees  and add feriepenge%

### PR DESCRIPTION
**Pull request by agvald**
Closes #460

### CODE REVIEW

**Comment by mladjo2505** [comment]
All code looks good so far.
The only thing that seems incorrect is putting the Feriepengeprosent to the employee instead on work relation. Ask Arnt.

**Comment by agvald** [comment]
@mladjo2505 
I talked with Arnt and he decided to leave the Feriepengeprosent on the accountplan

**Review by mladjo2505 — APPROVED** [review]

File: modules/salary/view/employeereport.php [hunk 1]
```diff
@@ -354,6 +376,74 @@ function print_sums($employee_id, $codes, $report) {
...
+
+print('
+<tr>
+  <td><b>sum innberettet</b></td>
+');
+foreach($codes as $c) {
+        printf('<td style="text-align: right">%s</td>', $_lib['format']->Amount($sum_account2[$c]));
+}
+
+if ($_lib['sess']->get_person('AccessLevel') > 1 ) {
```
Comment (agvald) [inline modules/salary/view/employeereport.php]: remove

File: modules/feriepenger/model/grid.class.php [hunk 2]
```diff
@@ -155,10 +154,11 @@ function gridPerson ($person)
 
         $myVar = "Prosentsats";
 
-        if ($res->$myVar != "" || $person == 1000)
+        if ($res->$myVar != "" || $person == 1000) {
```
Comment (agvald) [inline modules/feriepenger/model/grid.class.php]: Remove if if possible

File: modules/salary/view/employeereport.php [hunk 3]
```diff
@@ -401,6 +418,8 @@ function print_sums($employee_id, $codes, $report) {
         printf("<td style='text-align: right'>%s</td>", $_lib['format']->Amount($accountreport['amounts'][$c]));
     }
         print("<td><input type=\"checkbox\" name=\"DifferentYear\" checked disabled/></td>");
+        printf("<td>%s</td>", $_lib['format']->Amount($accountreport['AGA']));
+        printf("<td>%s</td>", $accountreport['Feriepengeprosent'] === NULL ? "0.00" : $accountreport['Feriepengeprosent']);
```
Comment (agvald) [inline modules/salary/view/employeereport.php]: Remove === NULL if possible

File: db/changes/173_add_aga_and_feriepenger_to_salaryreportaccount.sql [hunk 4]
```diff
@@ -0,0 +1,9 @@
+ALTER TABLE salaryreportaccount
+  ADD Feriepengeprosent decimal(16,2) DEFAULT NULL,
+  ADD AGAprosent decimal(16,2) DEFAULT NULL;
+
+ALTER TABLE accountplan
+  ADD Feriepengeprosent decimal(16,2) DEFAULT 10.2;
```
Comment (agvald) [inline db/changes/173_add_aga_and_feriepenger_to_salaryreportaccount.sql]: ADD Feriepengeprosent decimal(16,2) NOT NULL DEFAULT 10.2;

File: db/changes/173_add_aga_and_feriepenger_to_salaryreportaccount.sql [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+ALTER TABLE accountplantemplate
+  ADD Feriepengeprosent decimal(16,2) DEFAULT 10.2;
```
Comment (agvald) [inline db/changes/173_add_aga_and_feriepenger_to_salaryreportaccount.sql]: ADD Feriepengeprosent decimal(16,2) NOT NULL DEFAULT 10.2;

**Review by stoivo — APPROVED** [review]

**Pull request merged**